Gemfile.lock

```diff
@@ -144,7 +144,7 @@ GEM
     parallel (1.20.1)
     parser (3.0.1.1)
       ast (~> 2.4.1)
-    preservation-client (3.3.0)
+    preservation-client (3.3.1)
```

Comment:
this is required to get the new `validate_moab` method in the client that we need to call

Reply:
This ship has sailed, but this should have been a minor release since new functionality was added per semver.

Reply:
Doh

Reply:
I'll do another release to 3.4.0 and then we can update the gemfile here to use `~> 3.4.0`